Test/IDE/SchemaDesigner/SchemaOperationsSpec.hs

```diff
@@ -450,39 +336,12 @@ tests = do
                 (SchemaOperations.deleteColumn options inputSchema) `shouldBe` expectedSchema
         describe "update" do
             it "update a column's name, type, default value and not null" do
-                let tableAWithCreatedAt = StatementCreateTable CreateTable
-                            { name = "a"
-                            , columns = [
-                                    Column
-                                        { name = "updated_at"
-                                        , columnType = PTimestampWithTimezone
-                                        , defaultValue = Just (CallExpression "NOW" [])
-                                        , notNull = True
-                                        , isUnique = False
-                                        , generator = Nothing
-                                        }
-                            ]
-                            , primaryKeyConstraint = PrimaryKeyConstraint []
-                            , constraints = []
-                            , unlogged = False
-                            }
+                let tableAWithCreatedAt = StatementCreateTable $ tableAWithCreatedAtTable
 
-                let tableAWithUpdatedColumn = StatementCreateTable CreateTable
-                            { name = "a"
-                            , columns = [
-                                    Column
-                                        { name = "created_at2"
-                                        , columnType = PText
-                                        , defaultValue = Nothing
-                                        , notNull = False
-                                        , isUnique = False
-                                        , generator = Nothing
-                                        }
-                            ]
-                            , primaryKeyConstraint = PrimaryKeyConstraint []
-                            , constraints = []
-                            , unlogged = False
-                            }
+                let tableAWithUpdatedColumn = StatementCreateTable 
+                                                $ defCreateTable "a" 
+                                                $ pure $ setColumn "created_at2" PText
```

Comment:
It's more specific if we're using `[]` syntax here instead of `pure`

```suggestion
                let tableAWithUpdatedColumn = StatementCreateTable 
                                                $ defCreateTable "a" [ setColumn "created_at2" PText ]
```